cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -446,13 +446,13 @@ if(python)
     find_package(PythonLibs ${python_version} REQUIRED)
     if (tmva)
       find_package(NumPy REQUIRED)
+    else()
+      find_package(NumPy)
     endif()
   else()
     find_package(PythonInterp ${python_version})
     find_package(PythonLibs ${python_version})
-    if(tmva)
-      find_package(NumPy)
-    endif()
+    find_package(NumPy)
```

Comment:
Hi @dpiparo, this should be enough, but if it's required by something, we need to make sure it's found.
What part of ROOT needs to use Numpy other than TMVA?

Reply:
We need it for AsMatrix. But this must be optional: no numpy, no AsMatrix.

Reply:
Then it's fine as is.